exceptioncheck.ts

```diff
@@ -0,0 +1,10 @@
+const core = require("@actions/core");
+
+module.exports = {
+  find: (prefix, title) => {
+    const titleRegex = RegExp(`(^${prefix}:.*$)`);
+    if (titleRegex.test(title)) {
+      return prefix;
```

Comment:
why not return a boolean?

`return titleRegex.test(title);`

Reply:
We use the output of this check for logging (also I think original authors did it this way to be able to test the script easier in tests.js).

Reply:
got it, could just use the input for logging, but if it makes the test more straightforward it's fine